app/helpers/generic_work_helper.rb

```diff
@@ -5,6 +5,6 @@ def render_collection_links(solr_doc)
     links = collection_list.map do |collection|
       link_to collection.title_or_label, collection_path(collection.id)
     end
-    content_tag :span, t(:is_part_of) + ': ' + links.join(', ').html_safe
+    content_tag :span, t('sufia.collection.is_part_of') + ': ' + links.join(', ')
```

Comment:
I really dislike this too but html_safe wasn't doing the escaping, and I read that content_tag provides it so thought to just remove it and re-write the expected output to match what the helper is creating. :/

Same results this morning, the escaping still occurs with either html_safe or safe_join(links). 
```
content_tag :span, t('sufia.collection.is_part_of') + ': ' + links.join(', ').html_safe
(byebug) helper.render_collection_links(work_doc)
"<span>Is part of: &lt;a href=&quot;/collections/111&quot;&gt;Collection 111&lt;/a&gt;, &lt;a href=&quot;/collections/222&quot;&gt;Collection 222&lt;/a&gt;</span>"


content_tag :span, t('sufia.collection.is_part_of') + ': ' + safe_join(links)

(byebug) helper.render_collection_links(work_doc)
"<span>Is part of: &lt;a href=&quot;/collections/111&quot;&gt;Collection 111&lt;/a&gt;&lt;a href=&quot;/collections/222&quot;&gt;Collection 222&lt;/a&gt;</span>"
```
I'm not sure how to prevent that escaping.